### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by AllKinds**

### CODE REVIEW

File: pipeline.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 repoName: blue-stream/blue-stream-authenticator
 runTests: true
-testCommand:
+testCommand: npm test
 deployUponTestSuccess: true
 deployCommand:
 deployEnvironment: staging
```
Comment (omrishtam) [inline pipeline.yaml]: `deployEnvironment` should be empty, not with staging.

File: src/authentication/authentication.handler.ts [hunk 2]
```diff
@@ -27,7 +27,7 @@ export class AuthenticationHandler {
     static handleUser(req: Request, res: Response) {
         const userToken = jwt.sign(req.user, config.authentication.secret);
 
-        res.cookie('bs-token', userToken);
+        res.cookie('kd-token', userToken);
```
Comment (omrishtam) [inline src/authentication/authentication.handler.ts]: Why is the token hard coded and not from a configuration?

File: test.Dockerfile [hunk 3]
```diff
@@ -0,0 +1,11 @@
+FROM node:10.13-alpine
+ENV NODE_ENV=development
+WORKDIR /usr/src/app
+COPY ["package.json", "package-lock.json*", "npm-shrinkwrap.json*", "./"]
+RUN npm config set -g production false
```
Comment (omrishtam) [inline test.Dockerfile]: Delete this line

File: test.Dockerfile [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+RUN npm install --silent
+RUN npm install -g mocha
+COPY . .
+EXPOSE 50051
```
Comment (omrishtam) [inline test.Dockerfile]: Delete this line too

File: test.Dockerfile [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+EXPOSE 8080
```
Comment (omrishtam) [inline test.Dockerfile]: Change default port to 8080 in configuration.

File: test.Dockerfile [hunk 6]
```diff
@@ -0,0 +1,11 @@
+FROM node:10.13-alpine
```
Comment (omrishtam) [inline test.Dockerfile]: Should use `node:10.15-alpine`

Comment (omrishtam) [reply test.Dockerfile]: Also in `Dockerfile`

File: test.Dockerfile (hunk 5 above)
Comment (omrishtam) [reply test.Dockerfile]: Also update EXPOSE in `Dockerfile`

File: test.Dockerfile [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+ENV NODE_ENV=development
+WORKDIR /usr/src/app
+COPY ["package.json", "package-lock.json*", "npm-shrinkwrap.json*", "./"]
+RUN npm config set -g production false
+RUN npm install --silent
```
Comment (omrishtam) [inline test.Dockerfile]: Remove `--silent`

Comment (omrishtam) [reply test.Dockerfile]: Also in `Dockerfile`

File: test.Dockerfile [hunk 8]
```diff
@@ -0,0 +1,11 @@
+FROM node:10.13-alpine
+ENV NODE_ENV=development
+WORKDIR /usr/src/app
```
Comment (omrishtam) [inline test.Dockerfile]: Change `Dockerfile` ENV HOME to `/usr/src`

File: pipeline.yaml [hunk 9]
```diff
@@ -1,6 +1,6 @@
 repoName: blue-stream/blue-stream-authenticator
```
Comment (omrishtam) [inline pipeline.yaml]: Should be our repo name: `meateam/authenticator`

File: pipeline.yaml [hunk 10]
```diff
@@ -1,6 +1,6 @@
...
 runTests: true
-testCommand:
+testCommand: npm test
```
Comment (omrishtam) [inline pipeline.yaml]: Should be "npm test"

**Comment by Shahar-Y** [comment]
Approved by @AllKinds

**Pull request merged**